protobuf/drand/protocol.proto

```diff
@@ -24,10 +24,8 @@ service Protocol {
     // from all received keys and as well other information such as the time of
     // starting the DKG.
     rpc PushDKGInfo(DKGInfoPacket) returns (drand.Empty);
-    // Setup is doing the DKG setup phase
-    rpc FreshDKG(DKGPacket) returns (drand.Empty);
-    // Reshare performs the resharing phase
-    rpc ReshareDKG(ResharePacket) returns (drand.Empty);
+    // BroadcastPacket is used during DKG phases
+    rpc BroadcastDKG(DKGPacket) returns (drand.Empty);
```

Comment:
This is  a simplification of the processes - that was un-necessary to have two separate calls.